serverAkkaHttp/src/main/scala/RPCServer.scala

```diff
@@ -10,21 +10,20 @@ import akka.http.scaladsl.server.{ StandardRoute, Route }
 import scala.io.StdIn
 
 import wiro.models.Config
-import wiro.server.akkaHttp.RouteGenerators._
-import wiro.server.akkaHttp.RouteGenerators.BoxingSupport._
+import wiro.server.akkaHttp.{ Router => WiroRouter }
 
 class HttpRPCServer(
   config: Config,
-  controllers: List[GeneratorBox[_]],
+  routers: List[WiroRouter],
   customRoute: Route = reject
 )(implicit
   system: ActorSystem,
   materializer: ActorMaterializer
 ) {
   import system.dispatcher
 
-  val route = controllers
-    .map(_.routify)
+  val route = routers
+    .map(_.buildRoute)
     .foldLeft(customRoute) (_ ~ _)
```

Comment:
not relevant to this PR, but why is there a space between the two argument lists?